### PR TITLE
Fix vocal particle colours being blue for HARM2/HARM3 notes

### PR DESCRIPTION
**Pull request by thomeval**
This PR is to fix the vocal particle colours always being rendered blue, regardless of whether they are solo, HARM1 (blue), HARM2 (red), or HARM3 (yellow). 

vocalParticleColours

Note to the reviewer: This implementation matches the implementation for colouring the vocal notes themselves, which was clearly marked as a temporary solution (see VocalTrack.cs). As such, the hard-coded colour set should be replaced with colour profiles for vocals later on.

### CODE REVIEW

File: Assets/Script/Gameplay/Player/VocalsPlayer.cs [hunk 1]
```diff
@@ -60,6 +60,13 @@ public class VocalsPlayer : BasePlayer
 
             base.Initialize(index, player, chart, lastHighScore);
 
+            hud.Initialize(player.EnginePreset);
+            _hud = hud;
```
Comment (EliteAsian123) [inline Assets/Script/Gameplay/Player/VocalsPlayer.cs:64]: Was this change intentional? It looks like you just duplicated the hud initialization code from below.

**Review by EliteAsian123 — CHANGES_REQUESTED** [review]

File: Assets/Script/Gameplay/Player/VocalsPlayer.cs (hunk 1 above)
Comment (thomeval) [reply Assets/Script/Gameplay/Player/VocalsPlayer.cs:64]: Sorry about that, looks like I messed up when merging from the target branch.
